### DISCUSSION TITLE
Race Population by 2020 Censuns Blocks

### DISCUSSION THREAD

**Discussion by dldl1901s** [jakobzhao/geog495#63] [Final Project]
Link: https://dldl1901s.github.io/geog495_final_project/final.html
GitHub:  https://github.com/dldl1901s/geog495_final_project
Desc: This map shows race-specific population information in the block when the user clicks the desired Census Block on the map. You can also see the population density of King County at a glance.
![sample](https://github.com/dldl1901s/geog495_final_project/blob/main/img/sample.png?raw=true)

**Comment by LiamGearyUW** [comment]
Map is well-constructed, and population density is easy to understand when looking at it. A few things I would recommend changing: The sidebar ought to be a more neutral color. It being purple makes the legend harder to read. If this is for the average person to read, the geo code is unnecessary. Finally, the demographics section doesn't appear on the sidebar until you click on a block, which I find inelegant. Would probably look nicer if, at the very least, the legend remained in the same location and didn't shift to the bottom.

**Comment by TheAdonalsium** [comment]
Nice map. I really like how the shading is just for total population, but you can see total racial statistics if you look closer. I will say that you should maybe remove the oceans and lakes, bc it would make navigation much easier if I had some large physical features to recognize.

**Comment by aembar** [comment]
I really like the basemap chosen along with the color scheme for the total population(the regions really stand out). I would try changing the color of the sidebar since the legend is also different shades of purple and it blends in. I would also get hide the scrollbar that appears when you click on an area. I would also make the legend statistics be single spaced.

**Comment by NahomAgidew** [comment]
I like the color scheme you picked. An improvement I would like to suggest is after selecting a block, the population header moves way down to the bottom. You should move it up to use the extra space in between.
![image](https://user-images.githubusercontent.com/11695305/145707672-33b9b57d-677f-4635-85be-3b2b23868e5b.png)

**Comment by emilyphantastic** [comment]
Hi! I enjoy your topic and choice of colors. I like how you made the background map black and white, then had the data overlap it with shades of purple. A suggestion I have is to list what area the user is clicking on rather than providing a geo code. And possibly making the data layer more transparent so I know what city I'm clicking on because it's hard to tell/see. Also instead of setting all the data to 0 when the user clicks outside of the purple area, maybe have it say "N/A" instead because using "0" could be seen as misleading.

**Comment by meleakewubbie** [comment]
Your race population map is very informative. It's interesting to see which census blocks are densely populated. One issue is when I opened up the application and clicked a block some race statistics overlapped the legend. Other than that everything else looks perfect.
